### PR TITLE
Wolfy block placement

### PR DESCRIPTION
**Pull request by wolfyman411**
-Playtest island has been created
-Block placement has been added
-Blocks are removed from inventory when placed

The only thing left to do now is get the island heart producing blocks when it levels up, and to make sure it only accepts jelly dew.

### CODE REVIEW

File: Assets/Scripts/Player/Inventory/PlaceBlockHandler.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        else
+        {
+            Destroy(_activeBlockHelper);
+        }
+    }
+
+    private void PlaceBlockChecker()
+    {
+        //Check if the player presses m1 and make sure the backpack is not opened.
+        if (Input.GetButtonDown("Fire1") && !Inventory.Instance._isInBackpackMode)
```
Comment (CaseyWalshWarder) [inline Assets/Scripts/Player/Inventory/PlaceBlockHandler.cs]: You can place a block while the pause menu is open. You could replace the part after the && with !PlayerInteraction.Instance.IgnoreInputs

**Review by CaseyWalshWarder — COMMENTED** [review]
The icon for blocks is unity icon, so you can't see the number of items. The background shouldn't be black.

In the [JellyDew] prefab, jelly dew item identity is basic block rather than JellyDewItemIdentity.

Copy and paste the rigidbody settings from the berry item prefab to [PickupBlock]

On the [PickupBlock] prefab's PickupItem script, the pickup trigger field is assigned to the wrong collider. Assign it the one on the same gameObject.

The "GrassBlockData 1" scriptable object is unused, so delete it.

File: Assets/Scripts/Player/Inventory/PlaceBlockHandler.cs (hunk 1 above)
Comment (wolfyman411) [reply Assets/Scripts/Player/Inventory/PlaceBlockHandler.cs]: Made some changes, I tried !PlayerInteraction.Instance.IgnoreInputs by itself, and the player could still place blocks while the inventory is open. For now I have both checks, and it seems to be working fine.

**Review by CaseyWalshWarder — APPROVED** [review]
All set, good job!

**Pull request merged**